sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * Variable name change: BreakBlobLeaseOptions::breakPeriod -> BreakBlobLeaseOptions::BreakPeriod
 * CreateFromConnectionString now accepts unencoded blob name
 * TagConditions is changed to nullable
+* Variable name change: NextMarker is renamed to ContinuationToken, Marker is renamed to PreviousContinuationToken for ListContainersSegmentOptions, FindBlobsByTagsOptions and ListBlobsSegmentOptions
```

Comment:
For options, it is `Marker` to `ContinuationToken`.
For result, it is `Marker` to `PreviousContinuationToken`, and `NextMarker` to `ContinuationToken`.